Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -14,102 +14,100 @@ See the separate topics for a description and examples of each of these scenario
+-   `disruptionManagement`: The section for configuring management of daemon disruptions
+    -   `managePodBudgets`: if `true`, the operator will create and manage PodDisruptionBudgets for OSD, Mon, RGW, and MDS daemons. OSD PDBs are managed dynamically via the strategy outlined in the [design](https://github.com/rook/rook/blob/master/design/ceph/ceph-managed-disruptionbudgets.md). The operator will block eviction of OSDs by default and unblock them safely when drains are detected.
+    -   `osdMaintenanceTimeout`: is a duration in minutes that determines how long an entire failureDomain like `region/zone/host` will be held in `noout` (in addition to the default DOWN/OUT interval) when it is draining. The default value is `30` minutes.
+    -   `pgHealthCheckTimeout`: A duration in minutes that the operator will wait for the placement groups to become healthy (see `pgHealthyRegex`) after a drain was completed and OSDs came back up.
+        Operator will continue with the next drain if the timeout exceeds.
+        No values or `0` means that the operator will wait until the placement groups are healthy before unblocking the next drain.
+    -   `pgHealthyRegex`: The regular expression that is used to determine which PG states should be considered healthy.
+        The default is `^(active\+clean|active\+clean\+scrubbing|active\+clean\+scrubbing\+deep)$`.
+-   `removeOSDsIfOutAndSafeToRemove`: If `true` the operator will remove the OSDs that are down and whose data has been restored to other OSDs. In Ceph terms, the OSDs are `out` and `safe-to-destroy` when they are removed.
+-   `cleanupPolicy`: [cleanup policy settings](#cleanup-policy)
+-   `security`: [security page for key management configuration](../../Storage-Configuration/Advanced/key-management-system.md)
+-   `cephConfig`: [Set Ceph config options using the Ceph Mon config store](#ceph-config)
+-   `csi`: [Set CSI Driver options](#csi-driver-options)
 
 ### Ceph container images
 
-Official releases of Ceph Container images are available from [Docker Hub](https://hub.docker.com/r/ceph
-).
+Official releases of Ceph Container images are available from [Docker Hub](https://hub.docker.com/r/ceph).
 
 These are general purpose Ceph container with all necessary daemons and dependencies installed.
 
 | TAG                  | MEANING                                                   |
 | -------------------- | --------------------------------------------------------- |
-| vRELNUM              | Latest release in this series (e.g., *v17* = Quincy)      |
+| vRELNUM              | Latest release in this series (e.g., _v17_ = Quincy)      |
```

Comment:
How do the underscores render?

Reply:
Either bold or italics is fine with me as long as the rendering looks good, I just didn't realize this rendered as italics